modules/comments/base.php

```diff
@@ -277,18 +277,4 @@ public function set_comment_cookies( $comment_id ) {
 			setcookie( 'comment_author_url_'   . COOKIEHASH, esc_url($comment->comment_author_url), time() + $comment_cookie_lifetime, COOKIEPATH, COOKIE_DOMAIN );
 		}
 	}
-
-	/**
```

Comment:
@mdawaffe is out for awhile, so it'll be best to add the domain to the banned list and circle back later if needed.

Agree with adding any domains that fail to the banned list instead of removing the functionality outright.